docs/diagrams/UndoActivityDiagram.puml

```diff
@@ -0,0 +1,28 @@
+@startuml
+'https://plantuml.com/activity-diagram-beta
+
+start
+:User executes undo n;
+if () then ([n is not an integer])
+    :throw ParseException;
```

Comment:
```suggestion
    :Display invalid command message;
```
(off the top of my head I'm not sure if ParseException results in a different error message for CommandException, do confirm this!)